File: c++/insertionSort.cpp
#include <iostream>
using namespace std;

/* Driver code */
int main()
{
    int arr[] = {5, 2, 4, 6, 1, 3};
    int n = sizeof(arr) / sizeof(arr[0]);

    int i, j, temp;
    for (i = 0; i < n; i++)
    {
        for (j = n - 1; j > i; j--)
        {
            if (arr[j] < arr[i])
            {
                temp = arr[i];
                arr[i] = arr[j];
                arr[j] = temp;
            }
        }
    }
    for (i = 0; i < n; i++)
        cout << arr[i] << " ";
    cout << endl;
}

File: Python/InsertionSort.py
def insertionSort(arr, n):
    for i in range(0, n):
        j = n - 1
        while j > i:
            if arr[j] > arr[i]:
                arr[i], arr[j] = arr[j], arr[i]
            j -= 1

    for i in arr:
        print(i)


arr = [12, 11, 13, 5, 6]
n = len(arr)

insertionSort(arr, n)